src/Pages/Settings/Addons.elm

```diff
@@ -65,12 +68,116 @@ view : Model -> Document Msg
 view model =
     { title = "Settings.Addons"
     , body =
-        [ row [ Element.height fill, Element.width fill ]
-            (Components.VerticalNavSettings.view
-                model.route
-                ++ [ column [ Element.height fill, Element.width (fillPortion 6), spacingXY 5 7, Background.color Colors.background ]
-                        []
-                   ]
-            )
+        [ row [ Element.height fill, Element.width fill]
+            [ column[Element.height fill, Element.width fill, scrollbarY](Components.VerticalNavSettings.view model.route)
+            , column [ Element.height fill, Element.width (fillPortion 5), spacingXY 5 7, Background.color Colors.white, padding 40 , Font.size 24, Font.light]
+                        [ el[ Font.color (rgb255 18 178 231), Font.size 24, Font.light, paddingEach {top = 0, bottom = 30, left = 0, right = 0}](text "kodi.audioencoder")
```

Comment:
`kodi.audioencoder` is not the name we want for it, you can probably just use the correct english term for now.

Or we start looking at translating stuff.